hal.cabal

```diff
@@ -1,10 +1,10 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.33.0.
+-- This file has been generated from package.yaml by hpack version 0.34.5.
```

Comment:
I may look into de-noising the diffs a bit here.  I'll send some commits to pull in if I can, otherwise don't worry about it.